pkg/codegen/docs/gen.go

```diff
@@ -1011,11 +1015,13 @@ func (mod *modContext) genIndex(exports []string) string {
 	w := &bytes.Buffer{}
 
 	name := mod.mod
+	menu := false
 	if name == "" {
 		name = mod.pkg.Name
+		menu = true
```

Comment:
It'd be nice to have a comment here, why this flag is only set when the package name is blank.